src/main/java/BaseTest.java

```diff
@@ -0,0 +1,32 @@
+import PageFactory.HomePageFactory;
+import PageFactory.PasswordPageFactory;
+import org.openqa.selenium.WebDriver;
+
+import static driver.Driver.getDriver;
+import static driver.Driver.tearDown;
+
+public class BaseTest {
```

Comment:
move this class to src/test/java
so that this class with be stored together with other tests